_includes/header.html

```diff
@@ -1,8 +1,35 @@
 <header class="fx-row fx-row-between-xs fx-text-center">
-  <div class="fx-col fx-text-center"><a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
-      <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" /></a>
+  <div class="fx-col fx-text-center icon">
+    <a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
+        <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" />
+        <span>
```

Comment:
I'd not use span here. I think having a box element in an inline element is not valid html code. Please use <div>

Reply:
div instead?

Reply:
Yes.

Reply:
`<span>` is an inline element.  https://www.w3schools.com/tags/tag_span.asp (and elsewhere if you don't like W3Schools).